docs/source/_static/specs/formidable.js

```diff
@@ -1,546 +1,546 @@
 var spec = {
-    "basePath": "/api",
```

Comment:
is it me or are there just whitespace modifications here?

Reply:
I didn't touch this file. It was only regeneration of the doc